pkg/controller/node/node_controller.go

```diff
@@ -1173,33 +1173,4 @@ func (nc *Controller) maybeDeleteTerminatingPod(obj interface{}) {
 	if pod.DeletionTimestamp == nil {
 		return
 	}
-
```

Comment:
After this change the whole function is a no-op, and should be removed. This is rather big change, so I'd rather do it for 1.9 to give it more time to bake.

Reply:
+1
This shouldn't be merged for 1.8